lib/directus.ts

```diff
@@ -397,20 +402,22 @@ export const getDirectusClient = (): DirectusClient<any> & RestClient<any> => {
     return directusClient;
   }
 
-  if (!process.env.CMS_ENDPOINT) {
-    throw new Error("[server] CMS_ENDPOINT environment variable is not set");
+  if (!process.env.NEXT_PUBLIC_CMS_ENDPOINT) {
```

Comment:
I believe you don't need or want NEXT_PUBLIC here as this is server-only code. Should try updating it
https://nextjs.org/docs/app/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser